plugin/src/main/kotlin/tanvd/grazi/grammar/Typo.kt

```diff
@@ -38,6 +38,31 @@ data class Typo(val location: Location, val info: Info, val fixes: List<String>
             }
             return start in range
         }
+
+        fun isAtStartOfInnerElement(element: PsiElement): Boolean {
```

Comment:
A bit of documentation here would be nice.

Reply:
Done, a bit :)